rpm/xulrunner-qt5.spec

```diff
@@ -88,6 +88,8 @@ Patch39:    0039-sailfishos-embedlite-egl-Fix-mesa-egl-display-initia.patch
 Patch40:    0040-sailfishos-egl-Do-not-create-CreateFallbackSurface.-.patch
 Patch41:    0041-sailfishos-gecko-Fix-gfxPlatform-AsyncPanZoomEnabled.patch
 Patch42:    0042-sailfishos-gecko-Avoid-incorrect-compiler-optimisati.patch
+Patch43:    0043-sailfishos-gecko-Drop-static-casting-from-BrowserChi.patch
```

Comment:
However the patch name is`0042-sailfishos-gecko-Drop-static-casting-from-BrowserChi.patch`. Rebase issue?

Reply:
Also 0043-sailfishos-gecko-Avoid-incorrect-compiler-optimisati.patch doesn't match

Reply:
Amended commit and renamed rpm/0042-sailfishos-gecko-Avoid-incorrect-compiler-optimisati.patch to rpm/0043-sailfishos-gecko-Drop-static-casting-from-BrowserChi.patch